Add Default V2 to the theme list, remove Default and Bootstrap themes (#4344)
surveyjs/survey-library: docs/get-started-angular.md

## Configure Styles

> SurveyJS is shipped with two UI themes illustrated below: Modern and Default V2.

Reviewer: ```suggestion
SurveyJS ships with the Modern and Default V2 UI themes illustrated below.
```

![Themes in SurveyJS Library](images/survey-library-themes.png)

> Open the `angular.json` file and reference a style sheet that implements the required theme:

Reviewer: ```suggestion
Open the `angular.json` file and reference a style sheet that implements the theme:
```
